### ISSUE TITLE
Range error for unsigned 64-bit integers > 2**63 - 1

### ISSUE THREAD

**Issue by boydgreenfield** [Araq/Nim#728]
I muddled this issue and a printing issue that has since been fixed when reporting issue #723. I don't have the ability to reopen #723 directly, so flagging this here.

Nimrod raises a range error for 64-bit unsigned integers larger than `max(int64)`. This is an issue for hashing functions and others that may output anything in the range 0 .. 2**64 - 1. 

Here's a sample snippet that raises the error:
```
let x = 9223372036854775813'u64  # 2**63 + 5, fails here
echo(x)
```

And the error message:
```
temp.nim(9, 8) Error: number 9223372036854775813 out of valid range
```

Tested using 0.9.2 *and* the latest master branch.

**Comment by skyfex** [comment]
This is fixed now.

**Comment by boydgreenfield** [comment]
Closing. Confirmed that this is fixed in devel (0.10.3), but not on master (0.10.2).
